components/automate-ui/src/app/modules/infra-proxy/clients/clients.component.scss

```diff
@@ -1,5 +1,20 @@
 @import "~styles/variables";
 
+.search-create-container {
+  display: flex;
+  margin: 24px 0 18px;
+  flex-direction: row;
+
+  app-infra-search-bar {
+    flex-grow: 2;
+  }
+
+  chef-button {
+    margin: 0;
+    margin-left: 16px;
+  }
+}
```

Comment:
Author note:  The Search bar / Create button pairing has shown itself to be used over and over again within the infra page, meaning that instead of duplicating this CSS and pattern, It should be condensed into a single component upon further cleanup.  That will allow us to only maintain one set of css and html.